test/srt/test_srt_engine.py

```diff
@@ -0,0 +1,31 @@
+import json
+import unittest
+
+import sglang as sgl
+from sglang.test.test_utils import DEFAULT_MODEL_NAME_FOR_TEST
+
+
+class TestSRTBackend(unittest.TestCase):
+
+    def test_engine_runtime_consistency(self):
+        prompt = "Today is a sunny day and I like"
+        model_path = DEFAULT_MODEL_NAME_FOR_TEST
+
+        engine = sgl.Engine(model_path=model_path, random_seed=42)
+        out1 = engine.generate(prompt, {"temperature": 0})["text"]
+        engine.shutdown()
+
+        runtime = sgl.Runtime(model_path=model_path, random_seed=42)
+        out2 = json.loads(runtime.generate(prompt, {"temperature": 0}))["text"]
```

Comment:
```suggestion
        out2 = json.loads(runtime.generate(prompt, {"temperature": 0, "max_new_tokens": 8}))["text"]
```

Reply:
the results are constantly identical on my A100 hmm